core/src/main/java/com/amazon/opendistroforelasticsearch/sql/planner/logical/LogicalPlanNodeVisitor.java

```diff
@@ -53,4 +53,8 @@ public R visitRemove(LogicalRemove plan, C context) {
   public R visitEval(LogicalEval plan, C context) {
     return visitNode(plan, context);
   }
+
+  public R visitSort(LogicalSort plan, C context) {
+    return visitNode(plan, context);
+  }
```

Comment:
Could you help override this with pipelining Sort physical operator in `ElasticsearchIndex.implement`?

Reply:
sure.

Reply:
Thanks!